roles/supybot/templates/bb-supy.conf.j2

```diff
@@ -935,6 +935,103 @@ supybot.plugins.Owner.public: True
 ###
 supybot.plugins.Owner.quitMsg: 
 
+###
+# Determines whether this plugin is loaded by default.
+###
+supybot.plugins.Services: True
+
+###
+# Determines whether this plugin is publicly visible.
+#
+# Default value: True
+###
+supybot.plugins.Services.public: True
+
+###
+# Determines what nicks the bot will use with services.
+#
+# Default value:  
+###
+supybot.plugins.Services.nicks: bb-supy
+
+###
+# Determines what networks this plugin will be disabled on.
+#
+# Default value: QuakeNet
+###
+supybot.plugins.Services.disabledNetworks: QuakeNet
+
+###
+# Determines whether the bot will not join any channels until it is
+# identified. This may be useful, for instances, if you have a vhost
+# that isn't set until you're identified, or if you're joining +r
+# channels that won't allow you to join unless you identify.
+#
+# Default value: False
+###
+supybot.plugins.Services.noJoinsUntilIdentified: False
+
+###
+# Determines how many seconds the bot will wait between successive GHOST
+# attempts.
+#
+# Default value: 60
+###
+supybot.plugins.Services.ghostDelay: 60
+
+###
+# Determines what nick the 'NickServ' service has.
+#
+# Default value: 
+###
+supybot.plugins.Services.NickServ: NickServ
+
+###
+# Determines what password the bot will use with NickServ when
+# identifying as bb-supy.
+#
+# Default value: raip5aeR7O
```

Comment:
that is not our passwd is it?

Reply:
It is.  Why did it get written there? :(